Project-1/README.md

```diff
@@ -13,63 +13,226 @@ Note: The standard interpretation of the logical symbols - "∨", "∧", "→",
   ```
-  (a) (¬A→B)∨((A∧¬C)→B) 
-  (b) (A→B)∧(A→¬B)
-  (c) (A→(B∨C))∨(C→¬A) 
-  (d) ((A→B)∧C)∨(A∧D) 
+  (a) (¬A→B)∨((A∧¬C)→B) Tautology
+  A	B	C	((¬A → B) ∨ ((A ∧ ¬C) → B))
+F	F	F	T
+F	F	T	T
+F	T	F	T
+F	T	T	T
+T	F	F	T
+T	F	T	T
+T	T	F	T
+T	T	T	T
+
+  (b) (A→B)∧(A→¬B) Contingent
+  A	B	((A → B) ∧ (A → ¬B))
+F	F	T
+F	T	T
+T	F	F
+T	T	F
+
+  (c) (A→(B∨C))∨(C→¬A) Tautology
+  A	B	C	((A → (B ∨ C)) ∨ (C → ¬A))
+F	F	F	T
+F	F	T	T
+F	T	F	T
+F	T	T	T
+T	F	F	T
+T	F	T	T
+T	T	F	T
+T	T	T	T
+  (d) ((A→B)∧C)∨(A∧D) Contingent
+  A	B	C	D	(((A → B) ∧ C) ∨ (A ∧ D))
+F	F	F	F	F
+F	F	F	T	F
+F	F	T	F	T
+F	F	T	T	T
+F	T	F	F	F
+F	T	F	T	F
+F	T	T	F	T
+F	T	T	T	T
+T	F	F	F	F
+T	F	F	T	T
+T	F	T	F	F
+T	F	T	T	T
+T	T	F	F	F
+T	T	F	T	T
+T	T	T	F	T
+T	T	T	T	T
   ```
 	
 2. A _literal_ is an atomic formula or the negation of an atomic formula. We say a formula is in _conjunctive normal form_ (CNF) if it is the conjunction of the disjunction of literals. Find propositional logic formulas in CNF equivalent to each of the following:
   ```
   (a) (A→B)→C
+  (A∨C)∧(¬B∨C)
   (b) (A→(B∨C))∨(C→¬A)
+  Av~A
   (c) (¬A∧¬B∧C)∨(¬A∧¬C)∨(B∧C)∨A 
+ Bv~B
   ```
   
 3. Let V be the vocabulary of first-order logic consisting of a binary relation P and a unary relation F. Interpret P(x,y) as “x is a parent of y” and F(x) as “x is female.” Where possible define the following formulas in this vocabulary; where not possible, explain why: 
   
   ```
-  (a)  B(x,y) that says that x is a brother of y  
-  (b)  A(x,y) that says that x is an aunt of y  
-  (c)  C(x,y) that says that x and y are cousins   
-  (d)  O(x) that says that x is an only child  
-  (e)  T(x) that says that x has exactly two brothers 
+  Note that any definitions require that V has access to the connectives and quantifiers of first-order logic, which is not specified in the prompt. I have assumed that they are available, as otherwise there is no way to define any of the formulas in the following examples. 
+  
+  Also note that none of the gendered examples are definable if we allow for non-binary genders and if we read male or female as terms referring to sex rather than gender. In the first case, this would mean that we cannot assume that man and women are binary opposites to which these labels apply. In the latter, case, male or female unary relations will not refer to any of the gendered terms mentioned below.
+  
+  (a)  B(x,y) that says that x is a brother of y
+  B= ∃(x)∃(y)∃(z) (P(z,x)^P(z,y)^~F(x))
+  (b)  A(x,y) that says that x is an aunt of y
+  A(x,y)= ∃(x)∃(y)∃(z)∃(w) (P(z,y)^P(w,z)^P(w,x)^~P(x,y)^Fx).
+  (c)  C(x,y) that says that x and y are cousins
+  C(x,y): ∃(v)∃(w)∃(x)∃(y)∃(z) (P(v,x)^P(w,y)^P(z,v)^P(z,w)^~P(w,x)^~P(v,y))
+  (d)  O(x) that says that x is an only child 
+  O(x)= ∃(x)∃(y)(P(x,y)^~∃(z)P(x,z))
+  (e)  T(x) that says that x has exactly two brothers
+  T(x)= ∃(w)∃(x)∃(y)∃(z) ((P(w,x)^(P(w,y)^(P(w,z)^~F(y)^~(F(z))^~∃(v)(P(w,v))
```

Comment:
The above comments on your submission for "B" apply to the remaining expressions. They will all need to be corrected.

Reply:
These should all now have Universal scope and include non-equivalence relations